File: bee_3249.c
// Runtime Error
// *** stack smashing detected ***: terminated
// timeout: the monitored command dumped core

#include <stdio.h>
#include <string.h>

int main()
{
    int qntBatalhas, qntVitorias = 0;
    scanf("%d", &qntBatalhas);
    for (int i = 0; i < qntBatalhas; i++)
    {
        char batalha[200];
        scanf("%200s", batalha);
        if (strstr(batalha, "CD") == NULL)
        {
            qntVitorias++;
        }
    }
    printf("%d\n", qntVitorias);
    return 0;
}

File: bee_3249.py
# Death Knight Hero
def main():
    qntBatalhas = int(input())
    qntVitorias = 0
    for _ in range(qntBatalhas):
        batalha = input()
        if batalha.find("CD") == -1:
            qntVitorias += 1

    print(qntVitorias)

main()
